tpcx_bb/queries/q08/tpcx_bb_query_08_sql.py

```diff
@@ -0,0 +1,248 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Copyright (c) 2019-2020, BlazingSQL, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+
+from xbb_tools.cluster_startup import attach_to_cluster
+import cupy as cp
+import numpy as np
+import cudf
+
+from xbb_tools.utils import (
+    benchmark,
+    tpcxbb_argparser,
+    run_query,
+)
+
+cli_args = tpcxbb_argparser()
```

Comment:
I believe we don't need this anymore after the refactor and with the `config = tpcxbb_argparser()` in the main block

Reply:
You are completely right! Removed.